### DISCUSSION TITLE
Split settings into multiple screens

### DISCUSSION THREAD

**Discussion by rgmf** [OpenTracksApp/OpenTracks#663] [Ideas]
Preferences are getting bigger. With #659 we'll add new options.

I think that there are a lot of options in the preferences screen. We could split it into several screens. What do you all think?

**Comment by dennisguse** [comment]
I guess so. I am slightly scared by the complexity this might introduce. Having a fully configurable statistics screen (like positioning individual elements) would be a great feature, but it also introduces some maintenance costs.
But in general: yes, settings can be split.

PS: We can also check what we can remove (like default activity in settings and just use the last one the user had selected).

**Reply by rgmf** [reply]
With or without #659 I'm going to try to split settings into several screens I you don't mind.

Do you think there are more important things? Please, let me know if so. I have some time next week and I'd like to take something ;)

PS/ I like #659 but seem not easy...

**Reply by dennisguse** [reply]
Something I never took the time to work on is: https://github.com/OpenTracksApp/OpenTracks/issues/388
Somehow it feels weird that we use the Preferences to communicate the TrackRecordingId and if the recording is paused.
But this is might get very messy ;)
And it we have a workaround in the reset of the preference to not reset it if we are recording.

I have no idea, how to implement #659 properly, but it would be an awesome feature!

**Reply by rgmf** [reply]
> Something I never took the time to work on is: #388
> Somehow it feels weird that we use the Preferences to communicate the TrackRecordingId and if the recording is paused.
> But this is might get very messy ;)
> And it we have a workaround in the reset of the preference to not reset it if we are recording.

I take this. I'll see if I can find a good solution for this.
